sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -574,7 +574,10 @@ object ScalaJSPluginInternal {
         detector.detect(testFrameworks.value) map { case (tf, name) =>
           (tf, new ScalaJSFramework(name, env, classpath, logger, console))
         }
-      }
+      },
+      // Fix #1817
```

Comment:
I think a comment actually explaining what this does would be a good idea, so one doesn't need to have the issue tracker at hand...